client/components/auth/loginRegister-modal.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Modal from 'react-bootstrap/lib/Modal';
+import SigninForm from './signin-form';
+import SignupForm from './signup-form';
+import { Tabs, TabLink, TabContent } from 'react-tabs-redux';
```

Comment:
Thanks, I use this new source library since it is easier to map the tabs with different selections such as render a  signinForm or signupFrom. I can switch to react-bootstrap. I will fix everything  by tonight.